src/motor_hardware.cc

```diff
@@ -173,6 +177,24 @@ void MotorHardware::setWheelJointVelocities(double leftWheelVelocity, double rig
     return;
 }
 
+// Publish motor state conditions
+void MotorHardware::publishMotorState(void) {
+    ubiquity_motor::MotorState mstateMsg;
+
+    mstateMsg.header.frame_id = "motor_state";
+    mstateMsg.header.stamp    = ros::Time::now();
+    mstateMsg.header.seq      = 0;
```

Comment:
Don't put seq, its just confusing because ros will overwrite it anyway.

Reply:
Ok.  While testing after i pushed I saw this and was going to change it anyway.